core/src/integration-test/java/com/scalar/db/storage/jdbc/JdbcAdminImportTestUtils.java

```diff
@@ -559,7 +658,195 @@ private int getMajorVersion() {
     }
   }
 
+  private Map<String, Column<?>> prepareInsertColumnsWithGenericAndCustomValues(
+      TableMetadata tableMetadata, List<Column<?>> customColumns) {
+    Map<String, Column<?>> genericColumnValuesByName =
+        prepareInsertColumnsWithGenericValues(tableMetadata);
+    customColumns.forEach(column -> genericColumnValuesByName.put(column.getName(), column));
+
+    return genericColumnValuesByName;
+  }
+
+  private Map<String, Column<?>> prepareInsertColumnsWithGenericValues(
+      TableMetadata tableMetadata) {
+    return tableMetadata.getColumnNames().stream()
+        .map(
+            columnName ->
+                prepareGenericColumnValue(columnName, tableMetadata.getColumnDataType(columnName)))
+        .collect(Collectors.toMap(Column::getName, column -> column));
+  }
+
+  private Column<?> prepareGenericColumnValue(String columnName, DataType columnType) {
+    switch (columnType) {
+      case INT:
+        return IntColumn.of(columnName, 1);
+      case TEXT:
+        return TextColumn.of(columnName, "foo");
+      case BLOB:
+        return BlobColumn.of(columnName, "ABCDE".getBytes(StandardCharsets.UTF_8));
+      case FLOAT:
+        return FloatColumn.of(columnName, 1.2F);
+      case DOUBLE:
+        return DoubleColumn.of(columnName, 4.23);
+      case BIGINT:
+        return BigIntColumn.of(columnName, 101);
+      case BOOLEAN:
+        return BooleanColumn.of(columnName, true);
+      case DATE:
+        return DateColumn.of(columnName, LocalDate.of(1003, 7, 14));
+      case TIME:
+        return TimeColumn.of(columnName, LocalTime.of(5, 45, 33, 123_456_000));
+      case TIMESTAMP:
+        return TimestampColumn.of(columnName, LocalDateTime.of(1003, 3, 2, 8, 35, 12, 123_000_000));
+      case TIMESTAMPTZ:
+        return TimestampTZColumn.of(
+            columnName,
+            LocalDateTime.of(1003, 3, 2, 8, 35, 12, 123_000_000).toInstant(ZoneOffset.UTC));
+      default:
+        throw new AssertionError();
+    }
+  }
+
   public void close() throws SQLException {
     dataSource.close();
   }
+
+  @SuppressWarnings("UseCorrectAssertInTests")
+  public static class JdbcTestData implements TestData {
```

Comment:
I refactored the `JdbcAdminImportTestUtils` classes to store the test data for importable and non-importable tables in this object to improve readability.